src/main/java/org/mifos/connector/ams/zeebe/ZeebeUtil.java

```diff
@@ -39,4 +54,75 @@ public static <T> T zeebeVariable(Exchange exchange, String name, Class<T> clazz
         }
         return (T) content;
     }
+
+    public static LoanRepaymentDTO setLoanRepaymentBody(Exchange exchange) {
+        LoanRepaymentDTO loanRepaymentDTO=new LoanRepaymentDTO();
+        loanRepaymentDTO.setDateFormat("dd MMMM yyyy");
+        loanRepaymentDTO.setLocale("en");
+        loanRepaymentDTO.setPaymentTypeId(1);
+        loanRepaymentDTO.setTransactionAmount((BigDecimal) exchange.getProperty("amount"));
+        loanRepaymentDTO.setTransactionDate("02 August 2022");
```

Comment:
Why these values are hard-coded?

Reply:
Locale and payment id type will be same as pointed by Dhval. Date can be changed to current date.